rossanafmenezes/Kakelandet_My_Cookbook
# Create recycler view recipes list

Closes: #12 


### Description


This PR introduces the files and code to add a RecyclerView for the Recipe List app. 

Please note: 

For now, the screen can be accessed using the navigation drawer at the top left corner (not by clicking on the Recipes button on the home screen)

The RecyclerView is using a temporary file with dummy content (Recipes.kt) just for visual aid. 

### Testing instructions


- Run the app
- click on the Navigation Drawer
- Click on the Recipes menu item
- make sure the dummy content is loaded and the list is scrollable

### Images/gif


 <img src="https://user-images.githubusercontent.com/30724184/207946145-99c0f5c9-65eb-45c2-b1ed-424eb18cd4a0.gif" width="350"/>
